src/test/java/innohack/gem/entity/rule/rules/TikaContentRegexTest.java

```diff
@@ -0,0 +1,87 @@
+package innohack.gem.entity.rule.rules;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import innohack.gem.entity.GEMFile;
+import innohack.gem.entity.util.FileUtilForTesting;
+import innohack.gem.example.tika.TikaUtil;
+import innohack.gem.example.util.FileUtil;
+import innohack.gem.filegen.CsvFileGenerator;
+import innohack.gem.filegen.PdfFileGenerator;
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class TikaContentRegexTest {
+
+  @Test
+  public void testValidCheckTikaContent() throws Exception {
+
+    System.out.println("Testing testCheckTikaContent");
```

Comment:
since we are testing one file, consider removing Filewalk and looping logic to make unit test case more concise. otherwise good to go